src/sympc/approximations/reciprocal.py

```diff
@@ -31,7 +34,7 @@ def reciprocal(self, method: str = "NR", nr_iters: int = 10):
 
     if method == "nr":
         new_self = modulus(self)
-        result = 3 * exp(0.5 - new_self) + 0.003
+        result = 3 * exp(0.5 - new_self) + torch.tensor(0.003)
```

Comment:
Q: Can't you remove the tensor from here and do it in the "GradDiv":
```

if isinstance(x, (list, int, float)):
   x = torch.tensor(x)
```

Reply:
In gradsum you mean?

Reply:
Done